redbeams-model/src/main/java/com/sequenceiq/redbeams/domain/DatabaseConfig.java

```diff
@@ -0,0 +1,212 @@
+package com.sequenceiq.redbeams.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.ResourceStatus;
+import com.sequenceiq.cloudbreak.aspect.secret.SecretValue;
+import com.sequenceiq.cloudbreak.domain.Secret;
+import com.sequenceiq.cloudbreak.domain.SecretToString;
+
+@Entity
+public class DatabaseConfig {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "rdsconfig_generator")
```

Comment:
Replace all "RDS" names with "database" equivalents instead, e.g., "databaseconfig_generator", "databaseconfig_id_seq".

Reply:
Actually, you did do that in the schema already, so these items here just need to be updated.

Reply:
Hawk's eye!